index.html

```diff
@@ -114,6 +114,31 @@ <h2>A project for first-time contributions</h2>
         <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
         <!-- Keep one line of space above and below your card -->
 
+        <!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->
+        <div class="card">
+          <p class="name">Vansitha</p>
+          <p class="contact">
+            <i class="fab fa-twitter"></i>
+            <a href="https://twitter.com/vansitha12" target="_blank">@vansitha12</a>
+          </p>
+          <p class="about">I am a student programmer and currently focused on putting what I've learned into good use by contributing to open source projects. 😄</p>
+          <div class="resources">
+            <p>3 Useful Dev Resources</p>
+            <ul>
+              <li>
+                <a href="https://www.freecodecamp.org/" target="_blank" title="Best place to start your coding journey.">FreeCodeCamp</a>
+              </li>
+              <li>
+                <a href="https://www.w3schools.com/" target="_blank" title="Has good guides and examples on how to get things done for the technology your're using.">W3Schools</a>
+              </li>
+              <li>
+                <a href="https://stackoverflow.com/" target="_blank" title="Place where all programers goto when they encouter nasty bugs.">StackOverflow</a>
+              </li>
+            </ul>
+          </div>
+        </div>
+        <!-- ________*END TEMPLATE* Contributor card END ________  -->
```

Comment:
kindly change `END TEMPLATE` with your username

Reply:
Alright. Sorry about that!